core/templates/search/search_pages_results.html

```diff
@@ -251,49 +150,138 @@ <h4 class="term">{{ paginator.count }} results
+
+            {% block search_page_pagination_hidden_filters %}
+              {#### Filter preservation ####}
 
+              {# search text #}
+              <input type="hidden" name="proxtext" value="{{query.proxtext}}" />
+              <input type="hidden" name="proxdistance" value="{{query.proxdistance}}" />
+              <input type="hidden" name="ortext" value="{{query.ortext}}" />
+              <input type="hidden" name="andtext" value="{{query.andtext}}" />
+              <input type="hidden" name="phrasetext" value="{{query.phrasetext}}" />
 
-{% block search_results_box %}
-<div class="content search_results">
-  <h3 class="results">Results {{ start }}-{{ end }}</h3>
+              {# location #}
+              <input type="hidden" name="city" value="{{query.city}}" />
+              <input type="hidden" name="county" value="{{query.county}}" />
 
-<br/><br/>
+              {# date range and years #}
+              {# if there is a yearRange, it supercedes the date ranges #}
+              {% if request.GET.yearRange %}
+                <input type="hidden" name="yearRange" value="{{query.yearRange}}" />
+              {% else %}
+                <input type="hidden" name="date1" value="{{query.date1}}" />
+                <input type="hidden" name="date2" value="{{query.date2}}" />
+              {% endif %}
 
-<div class="row">
-{% for object in page.object_list %}
-  {% if forloop.counter0|divisibleby:4 and not forloop.first %}
-</div><div class="row">
-  {% endif %}
+              {# All the chosen LCCNs #}
+              {% for title in titles %}
+                <input type="hidden" name="lccn" value="{{ title.lccn }}" />
+              {% endfor %}
 
-    <div class="col-md-3">
-      <a href="{{ object.highlight_url }}" class="thumbnail">
-        <img src="{% thumb_image_url object %}" alt=""
-                class="highlight_words"
-                data-script_name="{{script_name}}"
-                data-id="{{object.url}}"
-                data-words="{{object.words|join:' '}}">
-        {% with object.issue.title as title %}
-          <p>
-            {{title.display_name}} ({{title.place_of_publication}})<br />
-            {{object.issue|label}}, {{object|label}}
-          </p>
-        {% endwith %}
-      </a>
-    </div>
-{% endfor %}
-</div><!-- /row -->
+              {# miscellaneous filters #}
+              <input type="hidden" name="language" value="{{query.language}}" />
+              <input type="hidden" name="frequency" value="{{query.frequency}}" />
+              <input type="hidden" name="sequence" value="{{query.sequence}}" />
+              <input type="hidden" name="sort" value="{{query.sort}}" />
+              <input type="hidden" name="rows" value="{{query.rows}}" />
+            {% endblock search_page_pagination_hidden_filters %}
 
-</div>
-{% endblock search_results_box %}
+            <input type="btn btn-primary" class="submit" value="GO" />
```

Comment:
I think class and type got switched around:  `<input type="submit" class="btn btn-primary" value="GO" />`

Reply:
I noticed because it looks like this right now, haha
<img width="488" alt="screen shot 2019-01-15 at 9 16 51 am" src="https://user-images.githubusercontent.com/2480492/51190177-3fc52300-18a7-11e9-8d2a-59f0c704030d.png">

Reply:
Oops! I'll fix that.